### PR TITLE
History feature Added.

### PR DESCRIPTION
**Pull request by sameep-s**
Videos Added into History.
Remove items from history.
Clear History.

### CODE REVIEW

File: src/util-reducers/data-reducer.js [hunk 1]
```diff
@@ -14,6 +9,30 @@ export function DataReducer(state_data, action) {
...
 
+        case "ADD_TO_HISTORY":
+            {
+                return {
+                    ...state_data, history: [...state_data.history, { ...action.payload.video, timeStamp: Date.now() }]
+                }
+            }
+
+        case "REMOVE_FROM_HISTORY": {
+            console.log(state_data.history);
```
Comment (rahulyadav139) [inline src/util-reducers/data-reducer.js:20]: Suggestion: remove console.log from here

File: src/components/CardVideoList/CardVideoList.jsx [hunk 2]
```diff
@@ -13,11 +13,15 @@ const CardVideoList = (video) => {
...
+
     return (
         <>
             <div className="container__card__videoListing m-1">
                 <Link to={`/videoPage/${_id}`}>
-                    <img src={thumbnail} alt="thumbnail" />
+                    <img src={thumbnail} alt="thumbnail" onClick={addToHistoryHandler} />
                 </Link>
                 <div className="card__videoListing__content">
                     <div className="card__videoListing_title flex mt-1">{title}</div>
```
Comment (rahulyadav139) [inline src/components/CardVideoList/CardVideoList.jsx:27]: Great you have used the BEM naming convention.

**Review by rahulyadav139 — APPROVED** [review]
Other than that everything looks great. The code is looking clean and it is readable too. Great work.

File: src/util-reducers/data-reducer.js (hunk 1 above)
Comment (sameep-s) [reply src/util-reducers/data-reducer.js:20]: left it by mistake, will keep that in mind.

**Review by suryxks — APPROVED** [review]
remove unnecessary space between two lines of code other than that code is readable and UI looks good ,good to go

**Pull request merged**